### PR TITLE
vHost target wait for enable file and use only for kvm hypervisor

### PR DESCRIPTION
**Pull request by giggsoff**
I can see "error set enable: /hostfs/sys/kernel/config/target/../enable no such file or directory", seems we
should wait a little for this file.
In general we expect vHost target only for kvm hypervisor so I put check for the hypervisor before we run logic for targets.

### CODE REVIEW

**Review by eriknordmark — APPROVED** [review]
LGTM

**Comment by eriknordmark** [comment]
@zededa-yuri can you also take a look?

File: pkg/pillar/cmd/volumemgr/updatestatus.go [hunk 1]
```diff
@@ -595,7 +595,7 @@ func doUpdateVol(ctx *volumemgrContext, status *types.VolumeStatus) (bool, bool)
 		}
 	}
 	if status.State == types.CREATING_VOLUME && status.SubState == types.VolumeSubStatePreparing {
-		if ctx.persistType == types.PersistZFS && !status.IsContainer() {
+		if ctx.useVHost && ctx.persistType == types.PersistZFS && !status.IsContainer() {
```
Comment (zededa-yuri) [inline pkg/pillar/cmd/volumemgr/updatestatus.go:598]: Maybe it's better to not to let the installation to proceed with the combination of `fs==zfs` and `hyper!=kvm`?

Comment (zededa-yuri) [reply pkg/pillar/cmd/volumemgr/updatestatus.go:598]: makes sense. You already have my LGTM

**Review by eriknordmark — APPROVED** [review]
LGTM

**Pull request merged**